src/solvers/solveCobraLPCPLEX.m

```diff
@@ -117,7 +117,10 @@
 if ~exist('minNorm','var')
     minNorm=0;
 end
-
+if ~printLevel && isunix

```

Comment:
As the value is turned off by default on Windows, there is no need for OS checking. The goal is to have less OS-dependent code.

The `if ~printLevel` works as `MATLAB` considers `0` as either `false` or an integer `0`. It is just very confusing to the reader of the code - hence the recommendation to change.